android/src/main/kotlin/com/abilia/system_settings_editor/VolumeSettingsHandler.kt

```diff
@@ -0,0 +1,66 @@
+package com.abilia.system_settings_editor
+
+import android.content.Context
+import android.media.AudioManager
+import io.flutter.plugin.common.MethodCall
+import io.flutter.plugin.common.MethodChannel
+
+class VolumeSettingsHandler(context: Context) {
+
+  private val audioManager: AudioManager = context.getSystemService(Context.AUDIO_SERVICE) as AudioManager
+
+  internal fun getAlarmVolume(): Double {
+    return audioManager.getStreamVolume(AudioManager.STREAM_ALARM) / getAlarmMaxVolume().toDouble()
```

Comment:
Im not sure that it is the ALARM audio channel we are using, and not NOTIFICATION...

Maybe make sure that it affects the "alarms" later

Reply:
Seems to be the RING stream, but I'll do some more testing.